### DISCUSSION TITLE
Adding shell:appsfolder as a folder

### DISCUSSION THREAD

**Discussion by Emub** [Hofknecht/SystemTrayMenu#323] [Q&A] [answered]
Hello. Is it possible to add the shell:appsfolder as a folder in systemtraymenu? I was wondering, since I'd like to add some apps and soforth to the menu

**Comment by Hofknecht** [comment, marked as answer]
this is related to #315 

found here c# code to get items from shell:appsfolder
https://stackoverflow.com/questions/57166533/how-can-i-retrieve-items-and-their-details-from-the-shellappsfolder-virtual-fol
(only a bit strange that correct answer has now upvotes)
needs a bit more analyze how we implement in detail, but we should then create issue from it somewhen.

of course as **workaround** you can select there your fav apps, make right click, generate shortcuts and then move them in a folder of STM

**Comment by Emub** [comment]
Thank you for looking into this, and thanks for the suggestion for the
workaround. I’ll do that. :-)

tor. 10. feb. 2022 kl. 15.23 skrev Markus Hofknecht <
***@***.***>:
